bgpd/rfapi/rfapi_vty.c

```diff
@@ -431,6 +431,14 @@ void rfapi_vty_out_vncinfo(struct vty *vty, struct prefix *p,
 		else
 			vty_out(vty, " label=%u",
 				decode_label(&bpi->extra->label[0]));
+
+		if (bpi->extra->num_sids) {
+			char buf[BUFSIZ];
+
+			vty_out(vty, " sid=%s",
+				inet_ntop(AF_INET6, &bpi->extra->sid[0], buf,
+					  BUFSIZ));
+		}
 	}
```

Comment:
not sure this piece of code should be modified.

Reply:
I see. that modification is not necessary while this contribution.
To simplify, I'll remove modification on rfapi.